src/sentry/releasehealth/sessions.py

```diff
@@ -1,14 +1,20 @@
 from datetime import datetime
-from typing import Optional, Sequence, Tuple
+from typing import Optional, Sequence, Set, Tuple, Union
 
 from sentry.releasehealth.base import (
+    CurrentAndPreviousCrashFreeRates,
     EnvironmentName,
     OrganizationId,
     ProjectId,
+    ProjectRelease,
     ReleaseHealthBackend,
     ReleaseName,
 )
-from sentry.snuba.sessions import _get_release_adoption, get_current_and_previous_crash_free_rates
+from sentry.snuba.sessions import (
+    _get_release_adoption,
+    check_has_health_data,
```

Comment:
I suggest renaming this function to lead with underscore, you will find some more places where it's being called. One is inside of `sentry.snuba.sessions` itself.

You could argue it doesn't matter as sentry.snuba.sessions already implies we're using teh sessions backend, but I think we should call through the service as much as possible in case we really want to implement dual-read-and-compare.